src/libraries/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -457,20 +457,15 @@
              Link="Common\System\Net\Http\HttpHandlerDefaults.cs" />
   </ItemGroup>
 
-<<<<<<< HEAD
   <!-- TODO-LLVM: This is not upstreamable because it makes the build runtime-specific. -->
   <ItemGroup Condition="'$(TargetPlatformIdentifier)' == 'wasi' and '$(RuntimeFlavor)' != 'CoreCLR'">
-    <Reference Include="System.Threading.Thread" />
-=======
-  <ItemGroup Condition="'$(TargetPlatformIdentifier)' == 'wasi'">
```

Comment:
Can we delete these ` '$(RuntimeFlavor)' != 'CoreCLR'` special cases now and take upstream verbatim? It would really like to avoid NAOT-LLVM conditionals outside of core runtime.

Reply:
Yes , we can